homework/01LessonOne.js

```diff
@@ -105,7 +106,7 @@ var notNotTrue;
 //
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: "and" is more strict and is used when all variables need to be "true". The "or" operator is more flexible.
```

Comment:
I think I was still lacking some understanding about these operators when I did this. Since the 'or' operator can be used to list a number of conditions and only one needs to be true, I guess I interpreted that in my mind as "flexible". Probably not the best way to describe it though.